packages/kmod-6.1-nvidia-r570/nvidia-tesla-tmpfiles.conf

```diff
@@ -0,0 +1,4 @@
+C /etc/drivers/nvidia-tesla.toml
+C /etc/drivers/nvidia-open-gpu.toml
+C /etc/drivers/nvidia-open-gpu-copy-only.toml
+C /etc/ld.so.conf.d/nvidia-tesla.conf
```

Comment:
Why did this line from r535 go away?
```
/etc/containerd/nvidia.env - - - - /usr/share/factory/nvidia/tesla/nvidia-path.env
```

Reply:
Per @arnaldo2792 it's no longer required since we changed the kmod directory structure.